integrations/azure/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,159 @@
+resources:
```

Comment:
Ask bogu what resources do we want to display for the defaults 

Maybe it should try to match the aws exporter defaults